packages/go/dawgs/graph/graph.go

```diff
@@ -128,7 +128,7 @@ func (s ID) Int64() int64 {
 }
 
 func (s ID) Sizeof() size.Size {
-	return size.Size(unsafe.Sizeof(s.Uint32()))
```

Comment:
Does the `ID` type on line 113 need to be updated to uint64?

Reply:
Yep, good catch!

Reply:
Updated to `uint64` base type.